src/core/bpf/allow_bind/allow-bind.c

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include <bpf/bpf_endian.h>
+#include <linux/bpf.h>
+#include <netinet/in.h>
+
+#include "maps.h"
+
+static inline int allow_bind_impl(
```

Comment:
use `__always_inline` instead of `inline`, don't leave it up to compiler to decide